apis/frontend/nosto-js/README.md

```diff
@@ -0,0 +1,75 @@
+# Nosto JS
+
+[Nosto JS](https://github.com/Nosto/nosto-js) is a wrapper for the client script, providing an easy way to interact with the Nosto service from your JavaScript or TypeScript applications.
+
+For more information about JS API, see [Our documentation](https://docs.nosto.com/techdocs/apis/frontend/js-apis).
+
+## Installation
+
+To install the package, use your preferred package manager:
+
+```bash
+yarn add @nosto/nosto-js
+# or
+npm install @nosto/nosto-js --save
+```
+
+## Modules
```

Comment:
Retain this is README instead. We need a clear distinction between the Github readme and this doc. Otherwise, it will be a duplicate information